Vagrantfile

```diff
@@ -132,6 +141,19 @@ Vagrant.configure("2") do |config|
       # Disable swap for each vm
       config.vm.provision "shell", inline: "swapoff -a"
 
+      if $kube_node_instances_with_disks
+        DISK_UUID = Time.now.utc.to_i
```

Comment:
what point of using this const instead variable?
it's warining:
```
warning: already initialized constant DISK_UUID
```

but const redefined as var (added `print "#{DISK_UUID} \n"` after DISK_UUID definition
```
1517310238
.../kubespray/Vagrantfile:144: warning: already initialized constant DISK_UUID
.../kubespray/Vagrantfile:144: warning: previous definition of DISK_UUID was here
1517310239
.../kubespray/Vagrantfile:144: warning: already initialized constant DISK_UUID
.../kubespray/Vagrantfile:144: warning: previous definition of DISK_UUID was here
1517310239
```

Reply:
Not sure to understand what you want me to do here. Can you please clarify?

Reply:
I'm not very good with ruby but when i using const (`DISK_UUID` ) i expect that value set once. In this steps const redefined in each loop iteration. 

If we need define const once then move definition inside `(1..$num_instances).each do |i|` loop.
If we need uniq id per vm then using var (`$dis_uuid`) and something else for generate random number.

Reply:
You're right, fixed.